### PR TITLE
Feature/scat 4358

### PR DESCRIPTION
**Pull request by ronin1972**

### CODE REVIEW

File: cat/CaT-service.yaml [hunk 1]
```diff
@@ -2060,6 +2061,255 @@ paths:
...
+            application/json:
+              schema:
+                type: array
+                items:
+                  allOf:
+                    - $ref: '#/components/schemas/DocumentSummary'
+                    - type: object
+                      description: Metadata relating to a template
+                      properties:
+                        id:
```
Comment (tommyb82) [inline cat/CaT-service.yaml:2087]: These properties seem to repeat those in `DocumentSummary` ..?

File: cat/CaT-service.yaml [hunk 2]
```diff
@@ -4691,13 +4950,17 @@ components:
       properties:
           id:
             type: string
+            description: ID for the document
           fileName: 
-            type: string  
+            type: string
+            description: The suggested filename for the document
```
Comment (tommyb82) [inline cat/CaT-service.yaml:4956]: Isn't this just `the filename` (for the client's info)?

**Review by tommyb82 — APPROVED** [review]
Looks good - not sure we need to override the `DocumentSummary` type for the response though.. 👍

File: cat/CaT-service.yaml (hunk 2 above)
Comment (ronin1972) [reply cat/CaT-service.yaml:4956]: Yes, they can use it to save the binary as the same name or a different name as they see fit.

File: cat/CaT-service.yaml (hunk 1 above)
Comment (ronin1972) [reply cat/CaT-service.yaml:2087]: Its just overriding the description to more appropriate text.

**Pull request merged**